Advocate/App.xaml.cs

```diff
@@ -46,6 +54,10 @@ protected override void OnStartup(StartupEventArgs e)
                 AllocConsole();
             }
 
+            // if we have a console, use it
+            if (GetConsoleWindow() != IntPtr.Zero)
```

Comment:
Unsure about the behaviour here when a parent process takes console output but doesn't have a console window